SwiftGen.playground/Sources/SwiftGenStoryboardEnumBuilder.swift

```diff
@@ -4,7 +4,8 @@ import Foundation
 
 public final class SwiftGenStoryboardEnumBuilder {
     private typealias Scene = (storyboardID: String, customClass: String?)
```

Comment:
Argh… seems like I discarded that change…

I had it like `StoryboardElement` or `UIElement` can't remember and the first parameter was actually `identifier` :/

Which name would you rather use? or none of the above?